code/mathematical_algorithms/reverse_number/reverse_number.c

```diff
@@ -0,0 +1,20 @@
+#include<stdio.h>
+
+int reverse_number(int n){
+	int ans = 0;
+	while (n != 0){
+		ans *= 10;
+		ans += n % 10;
+		n /= 10;
+	}
+	return ans;
+}
+
+int main(){
+	int n;
+	printf("Enter the number : ");
+	scanf("%d",&n );
```

Comment:
Remove the space before the closing parenthesis, and add a space after the comma.